recipe/meta.yaml

```diff
@@ -42,8 +42,8 @@ test:
         - test -f ${PREFIX}/lib/libdf.dylib  # [osx]
         - test -f ${PREFIX}/lib/libmfhdf.so  # [linux]
         - test -f ${PREFIX}/lib/libmfhdf.dylib  # [osx]
-        - conda inspect linkages -n _test hdf4  # [not win]
-        - conda inspect objects -n _test hdf4  # [osx]
+        - conda inspect linkages -p ${PREFIX} hdf4  # [not win]
+        - conda inspect objects -p ${PREFIX} hdf4  # [osx]
```

Comment:
Thanks @ocefpaf (and thank you for such a lightning fast response!)

I have updated this to use `${CONDA_DEFAULT_ENV}` instead. 

I am now investigating how to get the tests to pass.